### ISSUE TITLE
Unescaped unicode in response

### ISSUE THREAD

**Issue by MCON** [yajra/laravel-datatables#1034]
### Summary of problem or feature request

Can i send data to client with unescaped unicode json? It would economise several percent of bandwidth.

Here is the code i use:
Datatables::of($this->dealerMaintenancePartList)->make(true);

**Comment by redalpha01** [comment]
I'm not sure if this is what you mean but you can use ->rawColumns([$column1,]) on the columns where you don't want htmlentities to be applied to

**Comment by MCON** [comment]
Datatables::of($this->dealerMaintenancePartList)->make(true);
Produces for me:

`
{"draw":0,"recordsTotal":106503,"recordsFiltered":106503,"data":[{"id":1,"title":"\u0421\u041c\u0410\u0417\u041a\u0410 \u0414\u041b\u042f \u041a\u0410\u0420\u0414\u0410\u041d\u041d\u041e\u0413\u041e \u0412\u0410\u041b\u0410","model_code":"KUN26","model_name":"Hilux","part_number":"1307","price":"760.00","quantity":"0.10","catalogue_price_moscow":null,"maintenance_package_title":"\u0422\u041e 80.000 \u043a\u043c","updated_at":"2017-02-01 16:06:54","period_km":80000,"period_month":null},
`

I want it to be JSON_UNESCAPED_UNICODE
`
{
"id":1,
"title":"СМАЗКА ДЛЯ КАРДАННОГО ВАЛА",
"model_code":"KUN26",
"model_name":"Hilux",
"part_number":"1307",
"price":"760.00",
"quantity":"0.10",
"catalogue_price_moscow":null,
"maintenance_package_title":"ТО 80.000 км",
"updated_at":"2017-02-01 16:06:54",
"period_km":80000,
"period_month":null
}
`

My php and webserver are configured to send utf8. Thus \u0410\u041d\u041d\u041e\u0413\u041e  is a waste of bandwidth

**Comment by yajra** [comment]
Maybe try adding raw columns as suggested by @redalpha01 

```php
Datatables::of($this->dealerMaintenancePartList)
->rawColumns(['title'])
->make(true);
```

**Comment by MCON** [comment]
If i follow your advice i have:
Call to a member function getQuery() on null

is that an eloquent method rawColumns you advise me?

**Comment by yajra** [comment]
What version are you using?

**Comment by MCON** [comment]
"yajra/laravel-datatables-oracle": "6.*",

**Comment by MCON** [comment]
Well, i don't actually want to make any changes to composer package

**Comment by yajra** [comment]
rawColumns is not needed on v6. I think the issue here is the missing options on JsonReponse. I suggest you try to update [BaseEngine.php](https://github.com/yajra/laravel-datatables/blob/6.0/src/Engines/BaseEngine.php#L693) and tests the snippets provided above if it works.

If it does, then we can enhance the package as needed.

**Comment by MCON** [comment]
Yes. I've tested. it is fine now.
Can you add json flags to your package config file?

Also header should be sent:
Contnt-Type: application/json; charset=UTF-8

**Comment by yajra** [comment]
Submitted a PR, can you please test the PR if it works on your side? Thanks!

**Comment by MCON** [comment]
Actually we need it for version 6. We don't use ver7

**Comment by yajra** [comment]
My bad, added a new PR for 6.x. Thanks!

**Comment by MCON** [comment]
PR is fine. I suggest to accept it

**Comment by yajra** [comment]
PR merge and released on v6.29.0. But sadly tests are failing. Will fix it on next release. Thanks!
